### DISCUSSION TITLE
Component request: Message previews

### DISCUSSION THREAD

**Discussion by samarthpuri** [twilio-labs/paste#4097] [General] [answered]
Channels (RCS) and Content teams have been using phone mockups for message previews to showcase branding and rich content. Since other teams like Verify and SMS also use message previews, we're hoping to get standardized component in Paste to maintain consistency, especially with upcoming plans to add it in AMB

Content - [Figma](https://www.figma.com/design/Lg5Gw6cErsW0BZ5Z0VAR98/2023-Q4-Message-Preview?node-id=0-1&t=6rObUy9yohXVzOFP-1) 
RCS - [Figma](https://www.figma.com/design/4LoSo1G7dBh9ebjyvH5Git/2023-Q4-RCS?node-id=0-1&t=vOzpxWOaYJgi1RQX-1)

**Comment by serifluous** [comment, marked as answer]
Hey @samarthpuri ! You can reach out to Tony on Flex for this since he already maintains something similar for the Flex app store preview images.
